static/css/global.css

```diff
@@ -78,7 +78,7 @@ html,
 div#root {
 	min-height: 100vh;
 	background: var(--background);
-	overflow: hidden;
+	overflow: scroll;
```

Comment:
In a followup PR, I suggest changing it back to hidden, and instead, target only the styleguide page to enable overflow scroll.

Reply:
@aaronli03 I marked resolved so you can still merge